README.md

```diff
@@ -72,7 +73,13 @@ This employee also needs to pay taxes
            Tax Payable: £16.67
 
 Calculation rules:
- * Taxable income: Any amount of money earned above a gross annual salary of £11,000.00 is taxed at 20%
+ * Personal Allowance: Any amount of money earned up to £11,000.00.
```

Comment:
I'm not a fan of repeating all the rules every time. Why do you think this is necessary?